docs/hopsworksai/aws/upgrade.md

```diff
@@ -26,6 +26,9 @@ You need to **Stop** your cluster to start the upgrade process. Once your cluste
 
 ## Step 2: Add upgrade permissions to your instance profile
 
+!!! note
+    You can skip this step if you have created your instance profile following [this guide](../getting_started/#step-2-creating-instance-profile), where we add the upgrade permissions by default.
```

Comment:
I don't think this will work because people that followed the guide before we added the permissions will think that they followed the guide. I think we should say something like "you can skip if you already have permission X in Y"